scripts/test-e2e

```diff
@@ -2,36 +2,58 @@
 
 set -ueo pipefail
 
+cd $(dirname "$0")
+
+server="../dist/toxiproxy-server"
+
+benchmark() {
+  go test -bench=.. ../test/e2e -v
+}
+
+cli() {
+  ../dist/toxiproxy-cli $* 2>&1 | sed -e 's/^/[client] /'
+}
+
 wait_for_url() {
   curl -s --retry-connrefused --retry 5 --retry-delay 2 --retry-max-time 30 \
        --max-time 1 -L -I -X GET "${1}"
 }
 
 # Stop all background jobs on exit
-trap 'pkill -15 -f "dist/toxiproxy-server -proxy-metrics -runtime-metrics$"; pkill -15 -f "exe/endpoint$"' \
-     EXIT SIGINT SIGTERM
+function cleanup() {
+  sig="$1"
```

Comment:
```suggestion
  local sig="$1"
```

Bash variables are global even if declared in a function.

Reply:
nice. fixed.